src/parser/AttributeValueParser.java

```diff
@@ -0,0 +1,38 @@
+package parser;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class AttributeValueParser {
+    private Object value;
+    private static final Pattern ID_PATTERN = Pattern.compile(" Id :(.)+");
+    private static final Pattern COLOR_PATTERN = Pattern.compile("Color :(.)+");
+    private static final String ID = "Id";
+    private static final String COLOR = "Color";
+    private static final Integer SIX = 6;
+    private static final Integer EIGHT = 8;
+
+    public Object getValue(String valueName, String stringValue) {
+        switch (valueName) {
+            case ID:
+                Matcher idMatcher = ID_PATTERN.matcher(stringValue);
+
+                while (idMatcher.find()) {
+                    value = idMatcher.group().substring(SIX, EIGHT);
```

Comment:
Try to think and use matcher.group() without substring. Check documentation about group(index)/group().